packages/select/select.scss

```diff
@@ -101,6 +105,7 @@ $select-error-color--inverted: $feil--darkbg;
         background-color: var(--select-background-color);
         color: $select-text-color;
         color: var(--select-text-color);
+        cursor: pointer;
```

Comment:
[Burde ikke være pointer](https://adamsilver.io/blog/buttons-shouldnt-have-a-hand-cursor/)

Reply:
@piofinn argumenterte for det samme, men her føler jeg web i så lang tid har "overbrukt" pointer at man skaper forvirring ved å ikke bruke det. Selv om det kanskje er mer riktig, føler jeg selv at om et element som en knapp ikke har pointer på web, gir den uttrykk for at den egentlig ikke er klikkbar.

Etter en kjapp rundtur ser jeg at nettsteder som Gmail, Facebook, Twitter, GitHub og Outlook bruker konsekvent pointer på knapper. Samme gjelder designsystemer som Material Design og Carbon black. Selv har vi det også på Button, Toggle og Hamburger.

Skal ta opp dette som tema på neste designsystemforum - tenker at dette er noe vi må ha en uniform og gjennomgående tanke om i designsystemet.

Reply:
Hmm, det er sant det. Her er det nok mange som synder. Det er sikkert noe vi kan ta opp på neste designsystemforum.
Enig at det burde være gjennomgående for designsystemet i hvert fall :+1:

Reply:
tar vekk pointer effekten for nå, så avventer vi konklusjon fra designsystemforum før den gjeninnføres eller fjernes fra de andre komponentene som eventuelt ikke skal ha den :)